apps/nextjs-website/react-components/types/common/Common.types.ts

```diff
@@ -24,3 +24,5 @@ export interface CtaButtonProps extends Partial<ButtonProps> {
   readonly text: string;
   variant?: 'contained' | 'outlined';
 }
+
+export const hrefNoOp = 'javascript:void(0)';
```

Comment:
Why is this in `types`?

I would move it inside `components/common`, maybe creating a `consts.ts` or `helpers.ts` file inside the folder

Reply:
due to is only for Footer component, I've created a Footer.helpers.ts file for that